src/cursor/cur_backup.c

```diff
@@ -768,6 +784,8 @@ __backup_start(
     }
 
     cb->next = 0;
```

Comment:
Do we need to set `cb->cfg_allocated` to 0, or that's implied because the current list is NULL?

Reply:
The {{cb}} in backup_start is new so it has been zeroed already.